### DISCUSSION TITLE
What would you consider an acceptable CPU base level?

### DISCUSSION THREAD

**Discussion by memarkham303** [blakeblackshear/frigate#13914] [Ask A Question]
I have Frigate running on a Beelink EQ13 with two Coral TPUs.  It's monitoring 9 Amcrest cameras and doing a fine job.  Everything is great except that the CPU gauge in the lower left corner of the Frigate UI shows a **typical CPU consumption of about 30%**.

Would you consider that too high for normal operations?  I could spin-up a second Beelink EQ13 if warranted.

Thanks!

**Comment by NickM-27** [comment]
it's usually better to look at individual tasks and see what their CPU usage is, the cpu in the bottom left is on the whole host so can be higher based on other things running on the device.

**Comment by memarkham303** [comment]
Thanks for that. On the General Stats page, I see a few moments when the go2rtc process consumed about 10%, while recording ranges from 15% to 20%.
![Frigate CPU Processes](https://github.com/user-attachments/assets/c4e89569-cb24-48a8-8a90-16c44709c367)

It seems like it's probably not over-taxed and I can get away with one box.  I was concerned that 9 cameras might be too many.

**Comment by moandcompany** [comment]
Following the CPU utilization metric by itself can be misleading on some CPUs where clock speeds are dynamically controlled (e.g. burst/turbo frequency modes, etc).
